src/main/java/duke/tasks/ListCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.tasks;
+
+/**
+ * Represents a list command. This command handles the list input
+ * from users.
+ */
+public class ListCommand extends Command {
+    protected String list;
```

Comment:
I think this variable name is a little confusing as well, it has the same name as the method from TaskList. Perhaps you could change it to something more intuitive?